low-level/message_handlers/node_data_msg_handler.py

```diff
@@ -40,6 +41,10 @@
 
 from message_handlers.logging_msg_handler import LoggingMsgHandler
 from framework.utils.severity_reader import SeverityReader
+from framework.utils.store_factory import file_store
+
+# Override default store
+store = file_store

```

Comment:
Can you check if this does not have any side effect?

Reply:
For all node-related sensors, we are storing cache in file store in the SSPL data directory.
That's why I have made a change here.
I didn't see any side effects if we store cache in the file store. @thavanathan